cg-spec/editors_draft.html

```diff
@@ -2766,7 +2827,11 @@ <h2>Alternatives to referential opacity</h2>
           </li>
         </ul>
 
-      </section>
+      <p>Another alternative would be to make embedded triples referentially opaque for certain graphs or datasets only, reflecting the application centric character of the use cases they cover. The default semantics would then be referentially transparent while additional inferences on embedded triples could be suppressed for specific sets of triples.</p>
+
+      <p>A further option would be to tackle both controversial design choices at once by letting the shorthand syntax default to the main stream use case of <a href="https://github.com/w3c/rdf-star/issues/174">referentially transparent occurrences</a>. Possibly unexpected and complex behaviours would such be limited to the more involved and un-abbreviated syntax for the expert user.</p>
```

Comment:
```suggestion
      <p>Another alternative would be to make embedded triples referentially opaque only for certain graphs or datasets, reflecting the application-centric character of the use cases they cover. The default semantics would then be referentially transparent while additional inferences on embedded triples could be suppressed for specific sets of triples.</p>

      <p>A further option would be to tackle both controversial design choices at once by letting the shorthand syntax default to the mainstream use case of <a href="https://github.com/w3c/rdf-star/issues/174">referentially transparent occurrences</a>. Possibly unexpected and complex behaviors would thereby be limited to the more involved and un-abbreviated syntax for the expert user.</p>
```
The second paragraph does't seem to differ from the first.  #174 isn't a clear *anything*, neither problem statement nor solution plan nor comparison of options, and citing it here doesn't bring anything to the text.  I think the second paragraph should probably just be deleted (though I didn't do so in my `suggestion` above).

Reply:
Thanks for the edits to wording etc. The second paragraph does indeed introduce something new so it shouldn't be deleted. As this discussion is ongoing - we talked about implementing a second shortcut syntax specifically for referentially transparent occurrences, leaving the present shorthand for referentially opaque triples as is - it may of course change in the future.